src/operator/subgraph/mkldnn/mkldnn_fc.cc

```diff
@@ -156,7 +156,7 @@ void SgMKLDNNFCOp::Forward(const OpContext &ctx,
         int32_t *quantized_bias_ptr = cached_bias_.data().dptr<int32_t>();
         size_t bias_size = bias.shape().Size();
         #pragma omp parallel for num_threads(engine::OpenMP::Get()->GetRecommendedOMPThreadCount())
-        for (size_t i = 0; i < bias_size; ++i) {
+        for (int i = 0; i < static_cast<int>(bias_size); ++i) {
```

Comment:
Is the `static_cat` from `size_t` to `int` safety if the value of bias_size is large than the range of `int`?

Reply:
Already changed to `index_t`.